### PR TITLE
feat: swaggerhub integration 

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

Service Catalog SwaggerHub integration doc.

I added a section on how to add/find the API specs tab to link specs to services, because it took me some poking around to find it. It's not standard for these integration pages yet, but I think it makes sense to add it here.

https://konghq.atlassian.net/browse/DOCU-4092

### Testing instructions

Preview link: 
https://deploy-preview-8038--kongdocs.netlify.app/konnect/service-catalog/integrations/
https://deploy-preview-8038--kongdocs.netlify.app/konnect/service-catalog/integrations/swaggerhub/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/konnect/service-catalog/integrations/swaggerhub.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The SwaggerHub integration lets you connect SwaggerHub API specs directly to your Service Catalog services.
+
+## Prerequisites
+
+You need a SwaggerHub API key to authenticate your SwaggerHub account with {{site.konnect_short_name}}.
```
Comment (cloudjumpercat) [inline app/konnect/service-catalog/integrations/swaggerhub.md:17]:
```suggestion
You need a [SwaggerHub API key](https://app.swaggerhub.com/settings/apiKey) to authenticate your SwaggerHub account with {{site.konnect_short_name}}.
```
That link should take you to where you can get an API key in an account.

File: app/konnect/service-catalog/integrations/swaggerhub.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+## Authenticate the SwaggerHub integration
+
+1. From the **{% konnect_icon servicehub %} Service Catalog** in {{site.konnect_product_name}}, select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**.
```
Comment (cloudjumpercat) [inline app/konnect/service-catalog/integrations/swaggerhub.md:21]:
```suggestion
1. From the **{% konnect_icon servicehub %} Service Catalog** in {{site.konnect_product_name}}, select **[Integrations](https://cloud.konghq.com/service-catalog/integrations)**. 
```

File: app/konnect/service-catalog/integrations/swaggerhub.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## View SwaggerHub specs in Service Catalog
+
+You can map specs to a Service Catalog service and view them in {{site.konnect_short_name}}.
+
+1. From the **{% konnect_icon servicehub %} Service Catalog** in {{site.konnect_product_name}}, select **[Services](https://cloud.konghq.com/us/service-catalog)**. 
+2. Select a service, then open the **API Specs** tab.
+2. Select **Add API Spec**.
+3. Choose SwaggerHub as the **Source**, then pick your spec and name it.
+
+Once uploaded, you can view the rendered spec on the API Specs tab in structured, YAML, or JSON format, and download it in that format.
```
Comment (cloudjumpercat) [inline app/konnect/service-catalog/integrations/swaggerhub.md:36]:
```suggestion
Once it's uploaded, you can view the rendered spec on the API Specs tab in structured YAML or JSON format, and download it in that format.
```

**Review by cloudjumpercat — COMMENTED** [review]
Left a couple of comments! Let me know if you'd like another review.

Oh, I think it made sense where you put the instructions to map the SwaggerHub spec to the service, it is in a different spot than the other resources, so I think the instructions are helpful.

File: app/konnect/service-catalog/integrations/swaggerhub.md (hunk 3 above)
Comment (lena-larionova) [reply app/konnect/service-catalog/integrations/swaggerhub.md:36]: "Structured" is a separate option from YAML and JSON. It's the default - it's basically the rendered UI.

Comment (lena-larionova) [inline app/konnect/service-catalog/integrations/swaggerhub.md:36]:
```suggestion
Once it's uploaded, you can view the rendered spec on the API Specs tab in structured (UI), YAML, or JSON format, and download it in either YAML or JSON format.
```

**Review by cloudjumpercat — APPROVED** [review]
Changes look good to me!

File: app/konnect/service-catalog/integrations/swaggerhub.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The SwaggerHub integration lets you connect SwaggerHub API specs directly to your Service Catalog services.
+
+## Prerequisites
+
+You need a [SwaggerHub API key](https://app.swaggerhub.com/settings/apiKey) to authenticate your SwaggerHub account with {{site.konnect_short_name}}.
```
Comment (cloudjumpercat) [inline app/konnect/service-catalog/integrations/swaggerhub.md:17]:
```suggestion
You need a SwaggerHub API key to authenticate your SwaggerHub account with {{site.konnect_short_name}}.
```

**Pull request merged**